src/components/dashboard/lineChart.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+
```

Comment:
Can we add graph components to a separate folder in `common` so that it can be reused?

Reply:
Resolved this in commit - https://github.com/typeforslack/TypeForslack-Frontend/pull/6/commits/a43d07158c738d386b4535dc813878fb5cee658b